src/main/java/com/uid2/admin/vertx/service/SiteService.java

```diff
@@ -143,6 +121,55 @@ private void handleSiteList(RoutingContext rc) {
         }
     }
 
+    private static JsonObject createSiteObject(Site site, Map<Integer, List<LegacyClientKey>> clientKeys, List<LegacyClientKey> emptySiteKeys) {
```

Comment:
what do you think about potentially renaming to `createSiteJsonObject`? Since `site` is technically an object already

Reply:
Yeah I think that sounds reasonable.